spinalcordtoolbox/scripts/sct_download_data.py

```diff
@@ -77,19 +77,21 @@
     },
     "binaries_linux": {
         "mirrors": [
-            "https://github.com/spinalcordtoolbox/spinalcordtoolbox-binaries/releases/download/r20220516-2/spinalcordtoolbox-binaries_linux.tar.gz",
+            #  "https://github.com/spinalcordtoolbox/spinalcordtoolbox-binaries/releases/download/r20220516-2/spinalcordtoolbox-binaries_linux.tar.gz",
+            #  only for testing purposes
+            "https://github.com/spinalcordtoolbox/spinalcordtoolbox-binaries/releases/download/refs%2Fpull%2F4%2Fmerge-2981942540/spinalcordtoolbox-binaries_linux.tar.gz",
         ],
         "default_location": __bin_dir__,
     },
     "binaries_osx": {
         "mirrors": [
-            "https://github.com/spinalcordtoolbox/spinalcordtoolbox-binaries/releases/download/r20220516-2/spinalcordtoolbox-binaries_osx.tar.gz",
+            "https://github.com/spinalcordtoolbox/spinalcordtoolbox-binaries/releases/download/refs%2Fpull%2F4%2Fmerge-2981942540/spinalcordtoolbox-binaries_osx.tar.gz",
         ],
         "default_location": __bin_dir__,
     },
     "binaries_win": {
         "mirrors": [
-            "https://github.com/spinalcordtoolbox/spinalcordtoolbox-binaries/releases/download/r20220516-2/spinalcordtoolbox-binaries_windows.tar.gz",
+            "https://github.com/spinalcordtoolbox/spinalcordtoolbox-binaries/releases/download/refs%2Fpull%2F4%2Fmerge-2981942540//spinalcordtoolbox-binaries_windows.tar.gz",
         ],
```

Comment:
As a reminder, once the following PRs are merged:

* https://github.com/jqmcginnis/build_biomedia_stitching/pull/1
* https://github.com/spinalcordtoolbox/spinalcordtoolbox-binaries/pull/4

We will want to re-run the build/package workflows, then create suitably named releases (e.g. `r20221018`), then update the links so that we're not using the intermediate "refs%2Fpull%2F4%2Fmerge-2981942540" links.